test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/BasicAuthTests.cs

```diff
@@ -48,7 +48,7 @@ public BasicAuthTests(HttpIntegrationTestFixture fixture)
             _convertDataEnabled = convertDataConfiguration?.Enabled ?? false;
         }
 
-        [Fact]
+        [SkippableFact(Skip = "Auth Refactoring")]
```

Comment:
nit: no need to change for now, the base `[Fact(Skip = "Auth Refactoring")]` would work too